.github/workflows/nolints.yml

```
@@ -75,7 +75,7 @@ jobs:
           # github using a different username.
           git config --unset http.https://github.com/.extraheader
 
-      - name: update nolints.json and style-exceptions.txt
+      - name: file a PR to update nolints.json and style-exceptions.txt
```

Something which is not really clear from the description is what happens if a PR already exists. Reading `./scripts/update_nolints_CI.sh`, it seems that simply nothing happens, but personally I would have expected the PR to be updated.

---

Sure, let's make the wording more precise. :-) I'd rather ensure these PRs get merged quickly than spend time on improving the workflow for a scenario that hopefully never happens.
```suggestion
     - name: file a new PR to update nolints.json and style-exceptions.txt